server/common.js

```diff
@@ -177,6 +179,49 @@ var getDate = function () {
 }
 exports.getDate = getDate;
 
+/**
+* return the current date
+*
+* @return {date}
+*/
+var getDateObject = function(){
+    return new Date();
+}
+exports.getDateObject = getDateObject;
+
+/**
+* return the time in day hours minutes and seconds format
+*
+* @return {string}
+*/
+var getTime = function(time){
+    var date = new Date(time);
```

Comment:
It looks like most of the variables here are not modified, can we change those to `const`